README.md

```diff
@@ -1010,6 +1010,17 @@ skate.prop.boolean({
 
 Generally built-in properties will only return a definition containing `coerce`, `deserialize` and `serialize` options. They may also define a `deafult`, such as with the `boolean` property.
 
+Be aware that if you pass options into a property that you might be overriding default behaviour. With this in mind, it is up to you to make sure you callback or properly set anything that you override:
+
+```js
+const slot = skate.prop.slot();
+const prop = Object.assign(slot, {
```

Comment:
I think where our mindset might differ is that I treat the value of `skate.prop.slot()` **as a black box**. I definitely wouldn't expect `Object.assign(skate.prop.slot(), …)` to have any meaningful or predictable behaviour.

I get the sense that you're treating the value of `skate.prop.slot()` **not** a black box, and instead `skate.prop.slot()` is really just syntax sugar.

Do you think that's a fair assessment?